### DISCUSSION TITLE
Tools & CI meeting - October 28th, 2024 - Euro friendly edition

### DISCUSSION THREAD

**Discussion by spbrogan** [tianocore/edk2#6366] [General]
# Agenda

# Open Topics

# Call info

Microsoft Teams meeting
Join on your computer or mobile app
[Click here to join the meeting](https://teams.microsoft.com/l/meetup-join/19%3ameeting_ZDI2ZDg4NmMtMjI1My00MzI5LWFmYjAtMGQyNjUzNTBjZGYw%40thread.v2/0?context=%7b%22Tid%22%3a%2272f988bf-86f1-41af-91ab-2d7cd011db47%22%2c%22Oid%22%3a%2223af6561-6e1c-450d-b917-d9d674eb3cb6%22%7d)
Or call in (audio only)
+1 425-616-0754,,330776518# United States, Seattle
Phone Conference ID: 330 776 518#
Find a local number | [Reset PIN](https://mysettings.lync.com/pstnconferencing)
[Learn More](https://aka.ms/JoinTeamsMeeting) | [Meeting options](https://teams.microsoft.com/meetingOptions/?organizerId=23af6561-6e1c-450d-b917-d9d674eb3cb6&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&threadId=19_meeting_ZDI2ZDg4NmMtMjI1My00MzI5LWFmYjAtMGQyNjUzNTBjZGYw@thread.v2&messageId=0&language=en-US)

**Comment by os-d** [comment]
Continuing discussion from the previous meeting, incorporating feedback from https://github.com/tianocore/edk2/discussions/6364

**Comment by VivianNK** [comment]
GoogleTest topics:

- Gmocks as header files only: https://github.com/tianocore/edk2/pull/5950#issuecomment-2438970351
- Use of Static keyword: 
  - MU workaround: https://github.com/microsoft/mu_basecore/pull/908/files
  - Discussion thread: https://edk2.groups.io/g/devel/topic/108941574#msg120644
  - Last week's meeting discussed this: https://github.com/tianocore/edk2/discussions/6356
